ubg/src/game/VideoCard.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import {Typography} from '@material-ui/core';
+import {makeStyles} from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    width: '100%',
+    height: '100%',
+  },
+  videoName: {
+    'white-space': 'nowrap',
+    'overflow': 'hidden',
+    'text-overflow': 'ellipsis',
+  },
+}));
+
+/**
+ * @param {object} video A video object with video info
+ * @return {ReactElement} A card element for a video
+ */
+export default function VideoCard({video}) {
+  const classes = useStyles();
+  const link = video.link.replace('watch?v=', 'embed/');
+
+  return (
+    <Card className={classes.card}>
+      <CardMedia
+        component='iframe'
+        src={link}
+        image={link}
+        alt={video.title}
+        className={classes.media}
+        height='315'
+        width='560'
+      />
+      <CardContent>
+        <Typography gutterBottom variant='h5' component='h2'
```

Comment:
I don't remember if we said that if one prop is in another line then all should be.